### ISSUE TITLE
For android? 

### ISSUE THREAD

**Issue by agergo100** [FlashpointProject/launcher#282]
This is possible to make a android application for this program?

**Comment by Etaash-mathamsetty** [comment]
that would be awesome, i don't know how you are gonna get inputs (keyboard inputs) and the all the dependencies for this program on android.

**Comment by LindirQuenya** [comment]
No, it's **very** unlikely that this will ever occur. There are many reasons for this:
 - While android is Linux-based, it doesn't run docker (our long-term Linux plan), nor does it provide a whole host of Linux utilities/sockets/environment settings that we rely on for Flashpoint's Linux edition.
 - The inputs would be a nightmare. Can you imagine trying to get the keyboard to work?
 - Most Android devices are ARM. Literally every program that we use in FP is written for x86. Maybe, maybe with heavy modification, you could get QEMU working, but you might as well just virtualize a real Linux distro at that point.

In conclusion: no, it's never going to happen. There's a nightmare hiding behind every turn.
